benchmarks/array_transfer.py

```diff
@@ -7,9 +7,10 @@
 
 TYPES = ('int64', 'float64')
 
-def time_ak_array_transfer(N, trials, dtype, random, seed):
+def time_ak_array_transfer(N_per_locale, trials, dtype, random, seed):
     print(">>> arkouda {} array creation".format(dtype))
     cfg = ak.get_config()
+    N = N_per_locale * cfg["numLocales"]
```

Comment:
This one is actually meant to be N and not scale with the number of locales. (It's transferring between the single client and server, so we're limited by single client node.)

Reply:
ohhhh gotcha! thanks, I'll fix this